opm/simulators/flow/MissingFeatures.cpp

```diff
@@ -309,12 +301,9 @@ namespace MissingFeatures {
             "SKIP100",
             "SKIP300",
             "SKIPREST",
-            "SPECGRID",
             "SUMTHIN",
             "TEMP",
             "THCONR",
-            "TRACER",
-            "TRACERS",
             "VAPPARS",
```

Comment:
The Tracer model is not very well tested. I don't know the policy of this list, but since all the temperature related stuff are in this list, I assume that we should have confidence that the keyword is implemented correctly before removing it from the list. Do you agree? Maybe we should add a experimental list. "WARNING. Implementation of XXX is experimental"

Reply:
I'll leave them in for now. An experimental list can work, but I think it is better to simply have the experimental code print a warning itself. That way the warning is located with the code in question and will be removed when appropriate without having to remember to update some list, and the warning can be made as specific as possible.

Totally unsupported features have no associated code that can print a warning, which is why the list approach is necessary there.